app/components/ui/atoms/CalloutBox.js

```diff
@@ -2,16 +2,36 @@ import React from 'react'
 import styled from 'styled-components'
 import { Box } from 'grid-styled'
 import { th } from '@pubsweet/ui-toolkit'
+import Icon from './Icon'
 
-const CalloutBox = styled(({ enabled, ...rest }) => <Box {...rest} />).attrs({
-  mx: -3,
-  px: 3,
-  mb: 3,
+const Frame = styled(props => <Box {...props} />).attrs({
+  mb: 2,
 })`
   border: ${th('borderWidth')} ${th('borderStyle')} ${th('borderColor')};
   border-radius: ${th('borderRadius')};
-  border-color: ${({ enabled = true }) =>
-    enabled ? th('colorBorder') : 'transparent'};
+  border-color: ${th('colorBorder')};
+  position: relative;
 `
 
+const CloseButton = styled(Icon)`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  padding: ${th('space.2')};
+  svg {
+    stroke: ${th('colorFurniture')};
+    fill: ${th('colorFurniture')};
+  }
+`
+
+const CalloutBox = ({ closable, onClose, children, ...props }) => (
+  <Frame {...props}>
+    {closable && (
+      <CloseButton onClick={onClose} size={3}>
+        Cross
+      </CloseButton>
+    )}
+    <Box p={3}>{children}</Box>
```

Comment:
This perhaps deserves a wider conversation than this PR, but what do you guys think about using render props instead of higher order components?

One reason for this is readability. When an HOC is used (e.g. [in `FormSections.js`](https://github.com/elifesciences/elife-xpub/pull/257/files#diff-c54657ef098d458843cc24ce86e579f8R116)), it doesn't mean much to the person reading the file, unless they go and look into how the `CalloutBox` itself actually works.

Reply:
I'm not sure I follow what you mean by "higher order component".

Reply:
Oh I'm talking nonsense. This isn't a higher order component at all. It doesn't take in a component and return a component. :flushed:

Reply:
We did make a decision a while ago to favour render props, but, yes, not applicable here.